applications/RANSApplication/custom_processes/solving_strategies/scalar_co_solving_process.h

```diff
@@ -89,6 +93,17 @@ class ScalarCoSolvingProcess : public Process
         mRelaxationFactor = rParameters["relaxation_factor"].GetDouble();
         mMaxIterations = rParameters["max_iterations"].GetInt();
         mSkipIterations = rParameters["number_of_parent_solve_iterations"].GetInt();
+        mVtkOutputFrequency = rParameters["vtk_output_frequency"].GetInt();
+        mVtkOutputPrefix = rParameters["vtk_output_prefix"].GetString();
+
+        Parameters empty_parameters(R"({})");
+        if (!rParameters["vtk_output_settings"].IsEquivalentTo(empty_parameters))
+        {
+            KRATOS_INFO_IF(this->Info(), mEchoLevel > 0)
+                << "Adding VtkOutput.\n";
+            mpVtkOutput = unique_ptr<VtkOutput>(
```

Comment:
```suggestion
            mpVtkOutput = Kratos::make_unique<VtkOutput>(
```

Reply:
https://stackoverflow.com/questions/22571202/differences-between-stdmake-unique-and-stdunique-ptr-with-new